conf/cms/de/home.yaml

```diff
@@ -1,68 +1,58 @@
-bannerTopLeft:
+banners[0]:
   title: Sommer Essentials
   subtitle: Die wichtigsten Stücke des Sommer Trends 2016
-  linkText: Jetzt bestellen
-  linkHref: men
-  imageSmall: /assets/img/cms/leather1_small-4to3.jpg
-  imageMedium: /assets/img/cms/leather1_medium-4to3.jpg
-  imageLarge: /assets/img/cms/leather1_large-4to3.jpg
+  linkLabel: Jetzt bestellen
+  link: /de/men
+  backgroundImage: /assets/img/cms/leather1-4to3.jpg
```

Comment:
Why did we have to update our file based cms config files? I don't see the connection to the other changes, may be I'm missing something here?